_features/discoverable.html

```diff
@@ -2,34 +2,22 @@
 title: Discoverable
 slug: discoverable
 description: Are identifiable as applications and search engines can find them.
+usecases:
+- <p>Progressive Web Applications offer unique capabilities; as a developer, I want
+  my web applications to be promoted in search engines when the user is using
+  modern platforms.</p>
+- <p>As a search engine owner I want to highlight those applications with best UX
```

Comment:
Also, "with best UX" should have a comma after it:

> with best UX, promoting them to the upper positions."

But "best UX" is a bit misleading, since apps with an awful user experience can still publish web manifests. And "upper positions" is a bit oblique, even though I figured out what it meant (position in search results).

I'd reword this user story to something like, "As a search engine owner, I want to highlight apps that provide high-quality icons and other meta-data, promoting them in search results."